### ISSUE TITLE
[FEAT] Support Hover

### ISSUE THREAD

**Issue by Quorafind** [marcusolsson/obsidian-projects#52]
It will be nice to trigger popover when hover on a item, I tried to add this function but found it was diffcult to understand svelte callback. 😂

Some (maybe) useful code:

```typescript
export const hoverFile = (e: MouseEvent, item: TFile):void => {
    const targetEl = e.target as HTMLElement;

    if (!e.ctrlKey && !e.metaKey) return;

    app.workspace.trigger('hover-link', {
        event: e,
        source: 'multi-file-explorer-view',
        hoverParent: targetEl.parentElement,
        targetEl,
        linktext: item.name,
        sourcePath: item.path,
    });
}
```

**Comment by marcusolsson** [comment]
You mean like a link preview? Where you see the note in the popover? I can look into it!

**Comment by Quorafind** [comment]
![image](https://user-images.githubusercontent.com/13215013/194552745-ad77344d-357b-4855-b5a4-cc244aed54a8.png)

Yes, just like when you press ctrl+ hover on internal link, a link preview is shown.

https://user-images.githubusercontent.com/13215013/194553554-82a56511-2a6e-4aef-ac37-d53ce61f8416.mp4

**Comment by marcusolsson** [comment]
Available in 0.17.0.
